Train module refactor (#20)
WayScience/phenotypic_profiling_model: 2.train_model/README.md

# 2. Train Model

In this module, we train ML models to predict phenotypic class from cell features.

We train the models in [train_model.ipynb](1.train_model.ipynb).
We use [sklearn.linear_model.LogisticRegression](https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.LogisticRegression.html) for our machine learning models.

> A final model 

Reviewer: this looks like an incomplete sentence or typo?

Author: A typo, fixed in https://github.com/WayScience/phenotypic_profiling_model/pull/20/commits/c2a67f2d3c048099a41dcf4ad606c374181af490

- `C`: Inversely proportional to regularization strength.
We search over the following parameters: `[1.e-03, 1.e-02, 1.e-01, 1.e+00, 1.e+01, 1.e+02, 1.e+03]`

> A final model is trained with CP features, DP features, and merged (CP and DP) features.

Reviewer: ```suggestion
We train three final multiclass models with CP features, DP features, and merged (CP and DP) features.
```

Reviewer: active voice please! :)

Author: Ahhh! Of course

> A shuffled baseline model is trained for each of these feature types as well, with each column of the feature data being shuffled independently to create a shuffled baseline for comparison.

Reviewer: ```suggestion
We also train shuffled baseline models for each of these feature types as well, with each column of the feature data being shuffled independently to create a shuffled baseline for comparison.
```

> Each model is saved in [models/](models/).

Reviewer: ```suggestion
The notebooks save each model in [models/](models/).
```
